app/build_svc.py

```diff
@@ -2,6 +2,7 @@
 import glob
```

Comment:
In the import list, `import docker` should be after `import json` since it is a 3rd party package.
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Good catch, moved.